src/sentry/api/serializers/models/integration.py

```diff
@@ -13,16 +13,19 @@ def serialize(self, obj, attrs, user):
         provider = obj.get_provider()
         return {
             'id': six.text_type(obj.id),
+            'external_id': obj.external_id,
```

Comment:
Ooop, this should be `externalId`

Reply:
Pointed this out in standup, lets just use the `id` instead of external ID to identify what integration we're reinstalling